package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-associate-waf",
-  "version": "1.2.1",
+  "version": "1.2.1-rmd.1",
```

Comment:
We will need to produce a GitHub action to build and publish to our GitHub package repository, right?

Reply:
This repo is only intended to live until the change can be pushed upstream. My hope was to keep the things lightweight here and avoid needing to build and host the package unless there a constraint in a particular region that doesn't allow using the git link to resolve the package.